cli/src/test/java/com/devonfw/tools/ide/context/ProcessContextGitMock.java

```diff
@@ -175,13 +141,13 @@ public ProcessResult run(ProcessMode processMode) {
         Files.createDirectories(gitFolderPath);
         Path newFile = Files.createFile(gitFolderPath.resolve("update"));
         Files.writeString(newFile, this.now.toString());
-        this.exitCode = 0;
+        exitCode = ProcessResult.SUCCESS;
       } catch (IOException e) {
         throw new RuntimeException(e);
       }
     }
     this.arguments.clear();
-    return new ProcessResultImpl("git", command.toString(), this.exitCode, this.outs, this.errors);
+    return new ProcessResultImpl("git", "", exitCode, this.outputMessages);
```

Comment:
Why did you remove `command.toString()` and replace that with the empty String?

Reply:
Thanks for pointing that out. It's a mistake. I have adjusted it